py/setup.py

```diff
@@ -0,0 +1,40 @@
+import os
+import setuptools
+import shutil
+
+def get_requirements():
+    with open(u"requirements.txt") as fp:
+        return [x.strip() for x in fp.read().split("\n") if not x.startswith("#")]
+
+# Copies the schema in the module so that setuptools is able to
+# find the file and add it to the package.
+if os.path.isfile("../relay/event.schema.json"):
+    shutil.copyfile(
+        "../relay/event.schema.json",
+        "./sentry_data_schemas/event.schema.json"
+    )
+
+if os.path.isfile("../LICENSE"):
+    shutil.copyfile(
+        "../LICENSE",
+        "./LICENSE"
+    )
```

Comment:
Same as above. The main problem I have with this is we may not have the file at all and setup/build will just move on without even a warning.

Reply:
I can't really think of a simpler way to get this working so perhaps we just do the copyfile unconditionally adn leave it at that

Reply:
I don't like it either, but I could not find a way to reference from outside the root of the python package. 
Though you may have a point, in that I am not sure this will work properly (find the file) if we reference the package by github commit. Let me verify that.

Reply:
Unfortunately just doing the copy unconditionally would not work (added a comment about this):
- setup.py runs no matter whether we are building the distribution or importing tthe package. So depending on the phase the source file may or may not be there.
- When we build the distribution from source the source file is there, and by copying the file here, the json file will be in the package produced.
- Importing from source (whether gzipped or not) the source file is generally not there, but, when we built the distribution the file was copied where expected so it is available (but we must not try to copy). It works, I tried tot import the gzip
- Importing from pypi is supposed to work as above
- Importing from github would load the entire repo, (tried, it works) thus the source file is there and running the copy will work.

I think this was the intent for relay as well: https://github.com/getsentry/relay/blob/master/py/setup.py#L22-L27

I think a better solution may be to extend the setuptools sdist command. I tried and that part worked, though somehow the MANIFEST file is not copying all the files and I am not yet sure why. I need more investigation there.

Reply:
ok let's leave it like this